lib/internal/util/inspect.js

```diff
@@ -130,9 +130,58 @@ const typedArraySizeGetter = uncurryThis(
 
 let hexSlice;
 
-const builtInObjects = new Set(
-  ObjectGetOwnPropertyNames(global).filter((e) => /^[A-Z][a-zA-Z0-9]+$/.test(e))
-);
+// Make this list explicit instead of filtering over globalThis so that
+// this list is deterministic during the bootstrap regardless of:
+// a) Wether this file is required when the context is going to be
```

Comment:
```suggestion
// a) Whether this file is required when the context is going to be
```
(https://dictionary.cambridge.org/dictionary/english/wether 🐏 )